doc/bitcoin-conf.md

```diff
@@ -30,6 +30,21 @@ Network specific options can be:
 - placed into sections with headers `[main]` (not `[mainnet]`), `[test]` (not `[testnet]`) or `[regtest]`;
 - prefixed with a chain name; e.g., `regtest.maxmempool=100`.
 
+Network specific options take precedence over non-network specific options.
+If multiple values for the same option is found with the same precedence, the
+first one is generally chosen.
```

Comment:
with respect to "generally", do exceptions exist?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16448#discussion_r306685573

> with respect to "generally", do exceptions exist?

The main exception is settings like `-wallet` where multiple values are used. But in general the way settings are merged is insane and has lots of corner cases that should be cleaned up. #15934 has more information (look for for "strange merging behaviors" in the description).

I do think the description in this PR is basically at the right level of detail to be understandable and set users on the right path. Getting into a lot of exceptions and special cases here would probably not be helpful.

Reply:
Thanks! Will review #15934.

Reply:
Thanks, I'll leave this as is for now.